src/utils/input.js

```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
 
   return result
 }
+
+/**
+ * @param {number} valueLength - Length of input value
+ * @param {boolean} isRequired - If value is required
+ * @param {number} inputMaxLength - Minimum limit of length of the input value
+ * @returns {boolean}
+ */
+export const checkInputConstraints = (
+  valueLength,
+  isRequired,
+  inputMaxLength
+) => {
+  let isValid = false
+
+  if (inputMaxLength) {
+    isValid =
+      valueLength === inputMaxLength || (valueLength === 0 && !isRequired)
```

Comment:
You seems to compare strick equality with the length. Am I missing something?

As I read the code, the parameter should be named `expectedLength` or similar